### PR TITLE
Doc update core:duration

### PR DESCRIPTION
**Pull request by jpsiitonen**
Duration class example codes added/updated

### CODE REVIEW

File: sdk/lib/core/duration.dart [hunk 1]
```diff
@@ -193,41 +227,69 @@ class Duration implements Comparable<Duration> {
...
+  /// ```
   int get inHours => _duration ~/ Duration.microsecondsPerHour;
 
   /// The number of whole minutes spanned by this [Duration].
   ///
   /// The returned value can be greater than 59.
   /// For example a duration of three hours and 12 minutes
   /// has 192 minutes.
+  /// ```dart
+  /// const duration = Duration(hours: 3, minutes:12);
```
Comment (alexlindroos) [inline sdk/lib/core/duration.dart]: nit: space after minutes

File: sdk/lib/core/duration.dart [hunk 2]
```diff
@@ -193,41 +227,69 @@ class Duration implements Comparable<Duration> {
...
+  /// ```
   int get inMinutes => _duration ~/ Duration.microsecondsPerMinute;
 
   /// The number of whole seconds spanned by this [Duration].
   ///
   /// The returned value can be greater than 59.
   /// For example a duration of three minutes and 12 seconds
   /// has 192 seconds.
+  /// ```dart
+  /// const duration = Duration(minutes: 3, seconds:12);
```
Comment (alexlindroos) [inline sdk/lib/core/duration.dart]: nit: space after seconds:

File: sdk/lib/core/duration.dart (hunk 1 above)
Comment (jpsiitonen) [reply sdk/lib/core/duration.dart]: Updated

File: sdk/lib/core/duration.dart (hunk 2 above)
Comment (jpsiitonen) [reply sdk/lib/core/duration.dart]: Updated

**Review by alexlindroos — APPROVED** [review]

**Review by stepthomp — APPROVED** [review]

**Pull request closed without merging**